### ISSUE TITLE
Ninja 1.12.0

### ISSUE THREAD

**Issue by mayeut** [scikit-build/ninja-python-distributions#245]
Ninja 1.12.0 is available: https://github.com/ninja-build/ninja/releases/tag/v1.12.0

However the forked version used here https://github.com/Kitware/ninja is not yet available.

**Comment by Dominiquini** [comment]
Ninja 1.12.1 already available!

**Comment by MuellerSeb** [comment]
I seems that soon the forked version is not needed any more: https://github.com/ninja-build/ninja/pull/2506

**Comment by henryiii** [comment]
1.13 is now out, and the fork is no longer needed: #302

**Comment by jammm** [comment]
> 1.13 is now out, and the fork is no longer needed: [#302](https://github.com/scikit-build/ninja-python-distributions/pull/302)

When will this be available in pypi?

**Comment by henryiii** [comment]
See #314 (basically any time now). I think it might make sense to sync the release with CMake 3.1.

**Comment by jammm** [comment]
Thanks for your reply! Do you mean CMake 4.1?

**Comment by henryiii** [comment]
Yes, I've been constantly calling it 3.1, at least. :) I guess I've been doing too many 3.0 releases (cibuildwheel and pybind11). ;)
